packages/contracts/deploy/verification/99_conclude/00_save-contract-addresses.ts

```diff
@@ -47,5 +47,5 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
   );
 };
 export default func;
-func.tags = ['Conclude'];
+func.tags = ['New', 'Conclude'];
```

Comment:
What use does the `New` tag have?

Reply:
it will be used to differentiate between new fixture and update fixture in our testings